pkgs/development/compilers/graalvm/community-edition/ruby-installable-svm.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, graalvmCEPackages
+, openssl
+, javaVersion
+, musl
+, src
+, version
+, llvm-installable-svm
+}:
+
+graalvmCEPackages.buildGraalvmProduct rec {
+  inherit src javaVersion version;
+  product = "ruby-installable-svm";
+
+  extraBuildInputs = [
+    llvm-installable-svm
+    openssl
+  ];
+
+  preFixup = ''
+    patchelf $out/languages/ruby/lib/mri/openssl.so \
+      --replace-needed libssl.so.10 libssl.so \
+      --replace-needed libcrypto.so.10 libcrypto.so
+  '';
+
+  graalvmPhases.installCheckPhase = ''
+    echo "Testing TruffleRuby"
+    # Hide warnings about wrong locale
```

Comment:
This is just silencing, or a true fixup?

Reply:
It needs to have the locale setup for something, so it is a fix.